app/controllers/searches_controller.rb

```diff
@@ -1,2 +1,30 @@
 class SearchesController < ApplicationController
+  TWITTER_SEARCH_URI = "https://api.twitter.com/1.1/users/search.json?"
+
+  def index
+    if params[:search_twitter]
+      if params[:search_twitter] == ""
+        redirect_to root_path
+      else
+        redirect_to search_results_path("twitter", params[:search_twitter])
+      end
+    end
```

Comment:
This might be worth coming back to later to refactor after the params[:search_instagram] is implemented -- I think we could avoid the nested `if` blocks.